templates/projects/tasks_browse.html

```diff
@@ -179,6 +185,18 @@
                    <button type="button" class="btn btn-primary btn-sm" data-toggle="modal" data-target="#dateModal" data-info="ftime">Select date range</button>
                 </th>
                 {% endif %}
+                {% if 'gold_task' in filter_data.display_columns %}
+                <th>
+                    <div>
+                        <select id="gold_task">
```

Comment:
Add appropriate bootstrap classes so it looks like the other selects in the website.